CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 - Migrate `BloodSugarEntrySheet` to use `BaseBottomSheet`
 - Migrate `UpdatePhoneNumberDialog` to `BaseDialog`
 - Bump dagger to v2.43.2
-- Bump gradle to v7.5.1
```

Comment:
This is a different Gradle version. Let's add a new entry for the AGP v7.2.2 update.